doc/source/ray-contribute/docs.ipynb

```diff
@@ -413,11 +413,120 @@
     "And if you want to convert your notebook to a Python file, e.g. to test if your whole script runs without errors,\n",
     "you can use `--to py` instead.\n",
     "\n",
+    "(vale)=\n",
+    "\n",
+    "## How to use Vale\n",
+    "### What is Vale?\n",
+    "\n",
+    "[Vale](https://vale.sh/) checks if your writing adheres to the \n",
+    "[Google developer documentation style guide](https://developers.google.com/style). Currently,\n",
+    "it's only enforced on the Ray Data documentation.\n",
```

Comment:
```suggestion
    "It's only enforced on the Ray Data documentation.\n",
```

Reply:
There's a "Currently," on the preceding line